monster_waves/RandomGenerator.cpp

```diff
@@ -0,0 +1,15 @@
+#include "RandomGenerator.hpp"
+
+RandomNumber::RandomNumber(unsigned int minNumber, unsigned int maxNumber) : min(maxNumber),max(maxNumber),number(0)
+{
+   srand(static_cast<unsigned int>(time(nullptr)));
+}
+
+int RandomNumber::getNumber()
```

Comment:
@JakubMastalski Kompletnie nie wiem po co ta funkcja robi to co robi? Jaki jest jej cel?